### PR TITLE
Create first user automaticly

### LINKED ISSUE #64: create initial user automatically
At the moment after `docker-compose --env-file getuigenverhalen.env up` you need to create a user with curl to be able to use the user interface. It would be nice if the docker command already created a single user for you. The username could be part of the config file.

### PR DESCRIPTION
**Pull request by sverhoeven**
Uses WORKINGTITLECLOSEREADER_USER env var value as user name.

Fixes #64

### CODE REVIEW

**Comment by fdiblen** [comment]
> Uses WORKINGTITLECLOSEREADER_USER env var value as user name.

@sverhoeven  Could you also add this to somewhere in README.md?

**Review by fdiblen — CHANGES_REQUESTED** [review]
Neat!

Can you mention the environment variable somewhere in README.md?

**Comment by sverhoeven** [comment]
Will add to README once #68 or #61 has been merged, see #70

**Comment by sverhoeven** [comment]
Decided to not use env file anymore and use default in compose file and explain in readme how to overwrite default

File: README.md [hunk 1]
```diff
@@ -68,9 +68,35 @@ docker-compose build
...
+# (starting from the repo root directory)
+export EXPERIMENT=getuigenverhalen
+export FRONTEND_USER=myinitialusername
+docker-compose up
 ```
-$ curl -XPOST http://localhost:8080/users -d jspaaks   # note different port than what docs say
+
+### Add additional users
+
+If the existing user is not enough. You can add a user to the frontend with the following command
```
Comment (fdiblen) [inline README.md]:
```suggestion
If the existing user is not enough, you can add a new user to the frontend with the following command:
```

File: README.md [hunk 2]
```diff
@@ -68,9 +68,35 @@ docker-compose build
...
+(you can choose your own username by replacing `mynewusername` value in command)
+
+```shell
+export EXPERIMENT=getuigenverhalen
+export FRONTEND_USER=mynewusername
+docker-compose run usercreator
 ```
 
-Frontend should now be usable at [``http://localhost:8080``](http://localhost:8080).
+To add more repeat the command with differnent values for `FRONTEND_USER`.
```
Comment (fdiblen) [inline README.md]:
```suggestion
To add more users, repeat the command with different values for `FRONTEND_USER`.
```

File: ui/usercreator/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,4 @@
+FROM debian:buster
```
Comment (fdiblen) [inline ui/usercreator/Dockerfile:1]:
```suggestion
FROM debian:buster
ENV DEBIAN_FRONTEND noninteractive 
```

**Review by fdiblen — APPROVED** [review]
Clean and yummy.
:penguin:

**Pull request merged**